bundle/metadata/annotations.yaml

```diff
@@ -0,0 +1,14 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: STABLE_CHANNEL
+  operators.operatorframework.io.bundle.channels.v1: STABLE_CHANNEL
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: cockroachdb
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.3.0
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: helm.sdk.operatorframework.io/v1
+  operators.operatorframework.io.test.config.v1: tests/scorecard/
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
+
+  com.redhat.openshift.versions: 4.7-4.9
```

Comment:
This is not correct, why keep to 4.7-4.9. This will show the certified operator in 4.7 to 4.9 versions only

Reply:
removed